File: python/exponentiation.py
#!/user/bin/python

#   Exponentiation Algorithm in Python
#
# Coded by - Elton Fonseca
# 

def exponentiation(base, exponent):
    result = 1
    while exponent != 0:
        result *= base
        exponent -= 1
    return result

print exponentiation(2, 10)

File: javascript/exponentiation.js
/**
 *  Exponentiation Algorithm in JavaScript
 * 
 * Coded by - Elton Fonseca 
 **/

function exponentiation(base, exponent)
{
    result = 1;
    while(exponent != 0)
    {
        result *= base;
        exponent -= 1;
    }
    return result;
}

console.log(exponentiation(2, 10));